### PR TITLE
Add ignore-2.12.txt

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Please, after merge, add 2.11 to the test matrix (I didn't find it)

https://github.com/ansible-collections/overview/issues/45#issuecomment-813821100

### CODE REVIEW

File: changelogs/fragments/254-add_ignore_txt.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+trivial:
```
Comment (Qalthos) [inline changelogs/fragments/254-add_ignore_txt.yml:1]:
```suggestion
---
trivial:
```

The linter requires the start of document directive to be present

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: changelogs/fragments/254-add_ignore_txt.yml (hunk 1 above)
Comment (Andersson007) [reply changelogs/fragments/254-add_ignore_txt.yml:2]: @Qalthos committed, thanks

**Review by Qalthos — APPROVED** [review]

File: changelogs/fragments/254-add_ignore_txt.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+- Add ignore-2.12.txt (https://github.com/ansible-collections/ansible.netcommon/pull/254).
```
Comment (Qalthos) [inline changelogs/fragments/254-add_ignore_txt.yml:3]:
```suggestion
  - Add ignore-2.12.txt (https://github.com/ansible-collections/ansible.netcommon/pull/254).
```

The changelog linter is apparently _real_ strict

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Review by Qalthos — APPROVED** [review]

File: changelogs/fragments/254-add_ignore_txt.yml (hunk 2 above)
Comment (Andersson007) [reply changelogs/fragments/254-add_ignore_txt.yml]: ok, thanks

**Comment by Andersson007** [comment]
@Qalthos thanks for reviewing!

**Pull request merged**